.travis.yml

```diff
@@ -4,10 +4,8 @@ python:
   - "2.7"
 
```

Comment:
Good point. Will update to follow this example for maximum caching/speed https://github.com/dstufft/travis-example/blob/master/.travis.yml